includes/class-comment.php

```diff
@@ -216,6 +223,11 @@ public static function should_be_federated( $comment ) {
 			return false;
 		}
 
+		if ( is_single_user() && \user_can( $user_id, 'activitypub' ) ) {
```

Comment:
```suggestion
		if ( is_single_user() && \user_can( $user_id, 'publish_posts' ) ) {
```

Maybe we should keep checking `publish_posts`? If someone uses the blog-only mode, he might not care about setting up the caps properly?!?